_harp/services/ratecard.md

```diff
@@ -0,0 +1,16 @@
+# Rate card
+
+|    |    |
+|----|---:|
+|Graduate developer team|£500/week|
+|Graduate developer - follow-up work|£80/day|
+|Apprentice developer|£100/day|
```

Comment:
@iteles I was careful to say *from* in the case of developers and lead developers. This could easily be extended to apprentices, too. 

But it's helpful to give clients some indication of our daily rates. Advertising a lower bound on those rates probably makes sense.